homeassistant/components/anglian_water/entity.py

```diff
@@ -0,0 +1,26 @@
+"""Global representation of an anglian_water entity."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, NAME
+from .coordinator import AnglianWaterDataUpdateCoordinator
+
+
+class AnglianWaterEntity(CoordinatorEntity):
```

Comment:
```suggestion
class AnglianWaterEntity(CoordinatorEntity[..]):
```
Please add the coordinator type in the entity typing